js/classes/entities/midground/enemies/bunny.js

```diff
@@ -70,10 +70,15 @@ class Bunny {
     handleDeath() {
         this.action = "dying";
 
+        const pos = Vector.add(this.base.getCenter(), new Vector(0, -60));
```

Comment:
Minor, but maybe call this `dropPos` or something like that? Also, it may be helpful to make -60 a named constant.